packages/launcher/src/renderer/styles.css

```diff
@@ -1,9 +1,34 @@
+@font-face {
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 300;
+  src: local('Poppins Regular'), local('Poppins-Regular'),
```

Comment:
Is it OK to distribute these fonts according to the license please? Should we add their license somewhere in the repository as well?